frontend/src/static/css/v_authorship.scss

```diff
@@ -136,6 +132,18 @@
         margin-left: .5rem;
         text-decoration: none;
       }
+
+      .icons {
+        margin-right: 8px;
+        vertical-align: middle;
+      }
+
+      .fileTypeLabel{
```

Comment:
space before curly bracket, can you also help me to put up an issue to add this to style check?

Reply:
done